README.md

```diff
@@ -85,6 +91,7 @@ The supported metadata keys for omigami are (case insensitive):
 - "parent_mass",
 - "inchikey_smiles",
 - "inchikey_inchi"
+- "precursor_mz" or "pepmass"
```

Comment:
I think in our response we are using just 1 key for this (otherwise we would be sending duplicate data).
@carloszanella do you know which name we are currently using for the returned response?

Reply:
We can drop a note about the chosen name having both meanings or something maybe...